### PR TITLE
Milestone: Potfolio: Work and about section

### PR DESCRIPTION
**Pull request by Yakub-Ojo**
- Create the Work section
- Create the About section
- Style to fit the Figma design
- Commit my changes

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -86,3 +95,160 @@ main {
...
+
+button {
+  cursor: pointer;
+}
+
+.card-details {
+  align-items: center;
+  gap: 12px;
+  color: #172b4d;
+}
```
Comment (Tufahel) [inline style.css:143]: - Some issues I am mentioning below regarding this section. Kindly fix them.
- [ ] There shouldn't be spaces/padding on the right side of the images. I have marked red on the below picture. 
- [ ] I can see some extra dots on the left side of the list of items like `. Ruby on Rails, `. CSS`, etc. Kindly remove them.  You use list-style to fix this and consider reading the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) for how to do this.
- [ ] Use the given font for every `hx`, `p` and text items. You can browse the given template and see which font is needed for which section.

- Your Design:
- image

File: style.css [hunk 2]
```diff
@@ -86,3 +95,160 @@ main {
...
+  border-radius: 50%;
+  margin-top: 40px;
+  box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+}
+
+.skills {
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 285px;
+}
```
Comment (Tufahel) [inline style.css:254]: - [ ] I can see some extra dots on the left side of the list of skill items like `. Ruby on Rails, `. React.js`, etc. Kindly remove them.  You use list-style to fix this and consider reading the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) for how to do this.
- [ ] Kindly use the given font for every `hx`, `p` and text items. You can browse the given template and see which font is needed for which section.

- Your Design
- image

File: style.css [hunk 3]
```diff
@@ -86,3 +95,160 @@ main {
...
+
+.about-me {
+  background-image: url(./Icons/about-top.svg), url(./Icons/about_bg.svg);
+  background-repeat: no-repeat;
+  background-position: right -40px, 0 270px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
```
Comment (Tufahel) [inline style.css:177]: - [ ] Kindly use the given font for every `hx`, `p` and text items. You can browse the given template and see which font is needed for which section.

- Your Design:
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Yakub-Ojo  :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Commit messages are professional
✔️ Readme is professional.


## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (zieeco) [reply style.css:277]: - Hey, as requested by the previous reviewer, the list items should not have those dots. you can remove them by removing the list style.

``` 
ul,
li {
  list-style-type: none;
}
```

File: style.css (hunk 3 above)
Comment (zieeco) [reply style.css]: - As suggested by the previous reviewer, please, kindly use the correct fonts as used in the Figma design to improve your design.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Yakub-Ojo ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yakub-Ojo** [comment]
@zieeco , thanks for the review its really helpful, but concerning the dot showing on the I have removed it and the dots are not showing again I even summon the help of another reviewer to help me check from his end and he testify that the dots are not showing again, please kindly approve this as I am running out of time

Screenshot from 2023-03-25 23-32-29

**Review by jerryowusu — APPROVED** [review]
Hi @Yakub-Ojo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Nice design :heavy_check_mark: 
No linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**